.github/workflows/push-jsp-on-tag.yml

```diff
@@ -12,6 +12,15 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
+    - uses: actions/checkout@v2
```

Comment:
As its own list item with just this single line, this looks like a no-op. What am I missing? (Other than general GHA ignorance.)